Makefile

```diff
@@ -669,6 +669,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: kind-reset
+kind-reset: ## Destroys the "capi-test" kind cluster.
+	kind delete cluster --name=capi-test || true
+
+.PHONY: tilt-up
+tilt-up: kind-cluster ## Start tilt and build kind cluster if needed.
+	EXP_CLUSTER_RESOURCE_SET=true EXP_MACHINE_POOL=true tilt up
```

Comment:
Is there a reason we are setting these env values here. A user can set the feature gates in `tilt-settings.yaml`, right?

Reply:
I would also prefer having only one place where env vars / feature flags are enabled (tilt-settings.yaml) vs. overwriting them here hard-coded

Reply:
I think I just copied that over from my own set up. Just to clarify are we good to remove those env vars and just have it run `tilt-up`?

Reply:
I'm fine with that